src/desktop/apps/artist/client/views/related_artists.coffee

```diff
@@ -39,7 +39,6 @@ module.exports = class RelatedArtistsView extends ArtistArtworksView
         model: new Backbone.Model id: id
         modelName: 'artist'
         el: $el
-        href: artist.href
```

Comment:
These were being passed in to `FollowButton`, and used as the eventual `destination` in the auth modal. So, b/c we just want to end up where we started (which is the default in the auth modal if you don't specify `destination`), removed this everywhere.